gui/wxpython/tplot/frame.py

```diff
@@ -1460,7 +1460,7 @@ def __call__(self, event):
                 for a in event.artist.get_xdata():
                     try:
                         d = self.convert(a)
-                    except:
+                    except (ValueError, TypeError):
```

Comment:
All I was able to get from matplolib.dates.date2num was Indexerror and Valueerror